core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -1538,6 +1538,12 @@ class KafkaApis(val requestChannel: RequestChannel,
         (topic, Errors.NOT_CONTROLLER)
       }.toMap
       sendResponseCallback(results)
+    } else if (!config.deleteTopicEnable) {
+      val error = if (request.context.apiVersion < 3) Errors.INVALID_REQUEST else Errors.TOPIC_DELETION_DISABLED
```

Comment:
We also need to update DeleteTopicsRequest/DeleteTopicsResponse request/response classes to V3.

Reply:
Done!